lib/tests/test_mut_repo.rs

```diff
@@ -318,6 +318,33 @@ fn test_edit_initial() {
     );
 }
 
+#[test]
+fn test_edit_hidden_commit() {
+    // Test that MutableRepo::edit() edits a hidden commit and updates
+    // the view head ids.
+    let settings = testutils::user_settings();
+    let test_repo = TestRepo::init();
+    let repo = &test_repo.repo;
+
+    dbg!(repo.view().heads());
```

Comment:
nit: remove `dbg!()`?

Reply:
Ah, thanks for spotting that. It would be nightmare to me if this was accidentally submitted.